projects/plugins/jetpack/modules/theme-tools/infinite-scroll.php

```diff
@@ -14,20 +16,20 @@
  * @return null
  */
 function jetpack_load_infinite_scroll_annotation() {
-	if ( is_admin() && isset( $_GET['page'] ) && 'jetpack' == $_GET['page'] ) {
+	if ( is_admin() && isset( $_GET['page'] ) && 'jetpack' === $_GET['page'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Function loads theme specific IS file provided by Jetpack if possible.
 			$theme = wp_get_theme();
```

Comment:
I'm surprised phpcs didn't complain about this.
```suggestion
		$theme = wp_get_theme();
```